provider/fs/src/export/serializers/bincode.rs

```diff
@@ -59,7 +59,7 @@ impl AbstractSerializer for Serializer {
 }
 
 impl Serializer {
-    /// Creates a new serializer for [`bincode`].
+    #[doc(hidden)]
```

Comment:
Issue: I prefer avoiding `#[doc(hidden)]` except for (1) private APIs that need to be public for a legitimate reason like databake and (2) deprecated APIs for semver reasons, except that we should really be preferring `#[deprecated]` for that purpose. I don't think this one fits into either category unless you are intending to deprecate it. Is that your intent, and if so, why? Also, if you don't want people to use `Options`, you should make that deprecated as well.

tl;dr, to avoid additional debate, please revert the doc(hidden) changes here.